postcss.config.js

```diff
@@ -13,13 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-module.exports = {
-  plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-    cssnano: {
-      preset: 'default',
-    },
-  },
+const defaultPlugins = {
+  tailwindcss: {},
+  autoprefixer: {},
 };
+
+/* TODO: decide if we want it - it is little quicker but output tailwind.css is not minified therefore before pushing `npm run demo:css` need to be run*/
+const plugins = process.env.devLiveReloadMode
```

Comment:
:question: do you mean that for now, if we only run on 'watch' mode, the tailwind.css file won't be updated correctly or will be updated even if we had no changes in the style.css?

AFAIK the tailwind.css is a generated file, so we may probably exclude it from the source to avoid such issue. I don't like having generated file at the same place as source files, but as a 1st implementation, this will be better that keeping the file in git and have to deal with the source/generated files sync.

What do you think?

Reply:
yes, great idea
as discussed I've removed the generated file from the source 👍